crates/types/src/ball_filter_hypothesis.rs

```diff
@@ -1,22 +0,0 @@
-use std::time::{SystemTime, UNIX_EPOCH};
-
-use serde::{Deserialize, Serialize};
-
-use crate::kalman_filter::KalmanFilterSnapshot;
```

Comment:
Only work on this once we decided on the other review comment:

Remove `crate::kalman_filter` because it is dead code?